drivers/st7735/Kconfig

```diff
@@ -26,6 +26,12 @@ config MODULE_ST7789
     bool "ST7789 display driver"
     select MODULE_ST7735
 
+config ST77XX
```

Comment:
BTW, I have already a fix here and would solve it in a different way since the driver ST7735 is the selected component:
```diff
diff --git a/drivers/st7735/Kconfig b/drivers/st7735/Kconfig
index 40fa5643b7..0a27a4ef63 100644
--- a/drivers/st7735/Kconfig
+++ b/drivers/st7735/Kconfig
@@ -22,16 +22,16 @@ config HAVE_ST7735
     help
       Indicates that an ST7735 display is present.
 
-config MODULE_ST7789
-    bool "ST7789 display driver"
-    select MODULE_ST7735
-
 config HAVE_ST7789
     bool
-    select MODULE_ST7789 if MODULE_DISP_DEV
     help
       Indicates that an ST7789 display is present.
 
+config MODULE_ST7789
+    bool
+    depends on HAVE_ST7789
+    default y if MODULE_ST7735
+
 menuconfig KCONFIG_USEMODULE_ST7735
     bool "Configure ST7735 driver"
     depends on USEMODULE_ST7735
```
That is, of the board selects `HAVE_ST7789`, `MODULE_ST7789` is enabled by default if `MODULE_ST7735` is used. 

I have already locally an according extension for ST7796.

Reply:
That's better, I'll update when I'm back at the keyboard